src/hotspot/share/gc/shenandoah/shenandoahUtils.hpp

```diff
@@ -66,9 +66,10 @@ class ShenandoahTimingsTracker : public StackObj {
   const ShenandoahPhaseTimings::Phase   _phase;
   ShenandoahPhaseTimings::Phase         _parent_phase;
   double _start;
+  bool _should_aggregate_cycles;
 
 public:
-  ShenandoahTimingsTracker(ShenandoahPhaseTimings::Phase phase);
+  ShenandoahTimingsTracker(ShenandoahPhaseTimings::Phase phase, bool should_aggregate_cycles=false);
```

Comment:
Here and everywhere else, need whitespaces: `bool should_aggregate_cycles = false`